ytmusicapi/mixins/search.py

```diff
@@ -203,12 +203,100 @@ def search(self,
                                      category))
 
             if filter:  # if filter is set, there are continuations
-                request_func = lambda additionalParams: self._send_request(
-                    endpoint, body, additionalParams)
-                parse_func = lambda contents: parse_search_results(contents, type, category)
+
```

Comment:
What's the reason for this change?

Reply:
This change was made automatically when I used the command `pre-commit run`.

Reply:
In that case it's fine.